### PR TITLE
Temporary way of passing *some* environment variables to remote nodes

### PR DESCRIPTION
**Pull request by romain-intel**
This will be revisited with a more generalized solution where each individual decorator will be able to set the variables it wants to pass along

### CODE REVIEW

**Review by saikonen — APPROVED** [review]
as a temp solution this seems great. No worry about leaking user-specific envs with the whitelisting+`config_values` approach.

If I understood it correctly, then one gotcha that I thought of regarding extensions was that any env var that should be passable, needs to be accessed by `from_conf` before it is made available through `config_values`

What was the specific use-case for this improvement?

File: metaflow/plugins/aws/batch/batch.py [hunk 1]
```diff
@@ -249,6 +252,13 @@ def create_job(
             .environment_variable("METAFLOW_CARD_S3ROOT", CARD_S3ROOT)
             .environment_variable("METAFLOW_RUNTIME_ENVIRONMENT", "aws-batch")
         )
+
+        # Temporary passing of *some* environment variables. Do not rely on this
+        # mechanism as it will be removed in the near future
+        for k, v in config_values():
+            if k.startswith("METAFLOW_CONDA_") or k.startswith("METAFLOW_DEBUG_"):
+                job.environment_variable(k, v)
```
[the remark this answers is not in the document]
Comment (romain-intel) [reply metaflow/plugins/aws/batch/batch.py:260]: Values are serialized already using str or json.dumps so they should all be strings.

[the remark this answers is not in the document]
Comment (saikonen) [reply metaflow/plugins/aws/batch/batch.py:260]: oh, so it seems. Missed that part of `config_values`. Then disregard all type casting comments.

**Comment by romain-intel** [comment]
@saikonen : this is to make sure the netflix conda extension works cleanly here as well. And yes, things need to go through from_conf (any conf needs to go through that). Future improvement will probably use something like @environment to pass env vars (and some combo of the functionality from config_values). But for now this should be good.

**Pull request merged**